integration-tests/maven/src/test/resources/__snapshots__/CreateExtensionMojoIT/testCreateCoreExtension/dir-tree.snapshot

```diff
@@ -25,30 +14,25 @@ integration-tests/my-ext/src/main/java/org/acme/my/
 integration-tests/my-ext/src/main/java/org/acme/my/ext/
 integration-tests/my-ext/src/main/java/org/acme/my/ext/it/
 integration-tests/my-ext/src/main/java/org/acme/my/ext/it/MyExtResource.java
+integration-tests/my-ext/src/test/
+integration-tests/my-ext/src/test/java/
+integration-tests/my-ext/src/test/java/org/
+integration-tests/my-ext/src/test/java/org/acme/
+integration-tests/my-ext/src/test/java/org/acme/my/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/MyExtResourceTest.java
+integration-tests/my-ext/src/test/java/org/acme/my/ext/it/NativeMyExtResourceIT.java
 integration-tests/pom.xml
-pom.xml
+bom/
+bom/application/
+bom/application/pom.xml
```

Comment:
It seems I should add sorting to the dirtree snapshots output to avoid those unnecessary changes..

Reply:
https://github.com/quarkusio/quarkus/pull/16859